### PR TITLE
add #4 - view a list of all articles

### PR DESCRIPTION
**Pull request by AlistairHopson**

### CODE REVIEW

File: src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import "./Nav.css";
+
+export default function Nav() {
+  return (
+    <div className="navigation">
```
Comment (douglashellowell) [inline src/components/Nav/Nav.jsx:5]: `<nav>` is a nice symantic tag that could be used here!

**Comment by douglashellowell** [comment]
excellent stuff!

you're setting yourself up for an easier time by seperating out your components the way that you are (i.e: `ArticleCard`)

keep it up! 🚀

**Comment by douglashellowell** [comment]
Actually one thing to consider:

Try to limit the line-length in your app.
Humans find it hard to read a lot of text on one line.

have a look at the `max-length` css rule!

File: src/components/Nav/Nav.jsx (hunk 1 above)
Comment (AlistairHopson) [reply src/components/Nav/Nav.jsx:5]: Ah yes, good point!

**Comment by AlistairHopson** [comment]
Thanks, will do!

**Pull request merged**